src/opendr/engine/data.py

```diff
@@ -211,15 +211,32 @@ def __str__(self):
 class Image(Data):
     """
     A class used for representing image data.
+    OpenDR uses NCHW/RGB conventions
     This class provides abstract methods for:
     - returning a NumPy compatible representation of data (numpy())
+    - loading an input directly into OpenDR compliant format (open())
+    - getting an image into OpenCV- compliant format (opencv()) for visualization purposes
```

Comment:
**ToDo:** There's a space between _OpenCV-_ and _compliant_ that has to be removed.